src/views/waypoint/home/server.ts

```diff
@@ -0,0 +1,87 @@
+import slugify from 'slugify'
```

Comment:
Would like to discuss the introduction of this page, mainly to understand your decision making for context reasons. But that can come after the PR since this works!

Reply:
Sounds good, very on board with that! 💯  To clarify, `page` as in the `/waypoint` page, or `package` as in `slugify`?

Reply:
I didn't word that well! 😅  Please replace the word page `page` with `file`. The `src/views/waypoint/home/server.ts` file in general is what I'd like to learn more about. 😁

Reply:
ah sweet! Yeah for sure. Zoom's probably best, IIRC i did this cause of the combination of our `views / pages` organization with the NextJS requirement to split server-only imports into separate files, but would definitely love to dig into it a bit more and see if there's a different pattern we could adopt 🧐